src/zenml/orchestrators/local_docker/local_docker_orchestrator.py

```
@@ -0,0 +1,169 @@
+        pipeline: "BasePipeline",
+        pb2_pipeline: Pb2Pipeline,
+        stack: "Stack",
+        runtime_configuration: "RuntimeConfiguration",
+    ) -> Any:
+        """This method iterates through all steps and executes them in Docker.
+
+        Args:
+            sorted_steps: A list of steps in the pipeline.
+            pipeline: The pipeline object.
+            pb2_pipeline: The pipeline object in protobuf format.
+            stack: The stack object.
+            runtime_configuration: The runtime configuration object.
+        """
+        if runtime_configuration.schedule:
+            logger.warning(
+                "Local Docker Orchestrator currently does not support the"
+                "use of schedules. The `schedule` will be ignored "
+                "and the pipeline will be run immediately."
+            )
+        assert runtime_configuration.run_name, "Run name must be set"
+
+        from docker.client import DockerClient
+
+        docker_client = DockerClient.from_env()
+        image_name = runtime_configuration["docker_image"]
+        entrypoint = LocalDockerEntrypointConfiguration.get_entrypoint_command()
+
+        # Run each step
+        for step in sorted_steps:
+            if self.requires_resources_in_orchestration_environment(step):
+                logger.warning(
+                    "Specifying step resources is not supported for the local "
+                    "Docker orchestrator, ignoring resource configuration for "
+                    "step %s.",
+                    step.name,
+                )
+
+            arguments = (
+                LocalDockerEntrypointConfiguration.get_entrypoint_arguments(
+                    step=step,
+                    pb2_pipeline=pb2_pipeline,
+                    **{RUN_NAME_OPTION: runtime_configuration.run_name},
+                )
+            )
+            volumes = self._get_volumes(stack=stack)
+            user = None
+            if sys.platform != "win32":
```

Will this still work on Windows? If not, might want to raise an error instead

---

As far as I know, windows just doesn't use this concept of users to manage read/write permissions and works without setting anything. At least this is now in line with how we handled it in the case of local Kubeflow which also only sets the UID on Unix systems